CMSIS/system_stm32f4xx.c

```diff
@@ -34,9 +34,9 @@
      through STLINK MCO pin of STM32F103 microcontroller. The frequency cannot be changed
      and is fixed at 8 MHz. 
      Hardware configuration needed for Nucleo Board:
-     � SB54, SB55 OFF
-     � R35 removed
-     � SB16, SB50 ON */
+     �1�7 SB54, SB55 OFF
+     �1�7 R35 removed
+     �1�7 SB16, SB50 ON */
```

Comment:
How/why would this file have edits? Looking at the clock frequency? Even though this is a comment, I think we should revert this file to what it normally is.